### ISSUE TITLE
Custom matching regex

### ISSUE THREAD

**Issue by baubie** [shakyShane/grunt-cache-breaker#6]
If we could specify our own complete regex for matching it would make it more powerful.

**Comment by constructit** [comment]
Yes allowing wildcards like [grunt-cache-bust](https://github.com/hollandben/grunt-cache-bust) would save a lot of extra configuration time. Eg.

```javascript
match: [
    'app-components/**/*.js',
]
```
